### PR TITLE
docs: update Chinese getting started doc

### PR DESCRIPTION
**Pull request by hf400159**
### Description
Fixed punctuation issues in the installation document.
Update Chinese getting started docs, This PR is only responsible for the internationalization and synchronization of English changes from https://github.com/apache/apisix/pull/6929.
preview link: http://silly-stamp.surge.sh/zh/docs/apisix/next/getting-started/

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/getting-started.md [hunk 1]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+| Upstream    | 上游的作用是按照配置规则对服务节点进行负载均衡，它的地址信息可以直接配置到路由或服务上。               |
+| Admin API   | 用户通过该组件可以控制 APISIX 实例的 API。                                                         |
+
+## 前提条件
+
+在开始使用 APISIX 之前，请确保你已经安装以下应用：
+
+- [Docker](https://www.docker.com/) 和 [Docker Compose](https://docs.docker.com/compose/)。
+- [curl](https://curl.se/docs/manpage.html) 用于测试 API。你也可以使用 [Hoppscotch](https://hoppscotch.io/) 之类的工具。
+- 本文使用的上游服务是 [httpbin.org](https://httpbin.org)，你可以使用它进行测试。它是一个返回服务，这意味着它将返回我们在请求中传递的参数。
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:75]:
```suggestion
- 本文使用的上游服务是 [httpbin.org](https://httpbin.org)，你可以使用它进行测试。这是一个返回服务，它将返回我们在请求中传递的参数。
```

File: docs/zh/latest/getting-started.md [hunk 2]
```diff
@@ -68,51 +109,75 @@ curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
...
-git clone https://github.com/apache/apisix-docker.git
-# 将当前的目录切换到 apisix-docker/example 路径下
-cd apisix-docker/example
-# 运行 docker-compose 命令，安装 Apache APISIX
-docker-compose -p docker-apisix up -d
+sudo fuser -k 9443/tcp
 ```
 
-> Apache APISIX 已支持 ARM64 架构。如果您正在使用 ARM64，请在最后一步执行 `docker-compose -p docker-apisix -f docker-compose-arm64.yml up -d`。
+如果 Docker 容器不能正常运行，你可以通过以下命令检查日志以诊断问题：
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:165]:
```suggestion
如果 Docker 容器不能正常运行，你可以通过以下命令检查日志进行问题诊断：
```

File: docs/zh/latest/getting-started.md [hunk 3]
```diff
@@ -68,51 +109,75 @@ curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
...
 
-下载所需的所有文件将花费一些时间，这取决于您的网络，请耐心等待。
+```bash
+docker logs -f --tail $<container_id>
+```
+
+:::
 
-下载完成后，在运行 Docker 的宿主机上执行 `curl` 命令访问 Admin API，根据返回数据判断 Apache APISIX 是否成功启动。
+安装完成后，你可以在运行 Docker 的宿主机上执行 `curl` 命令访问 Admin API，根据返回数据判断 APISIX 是否成功启动。
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:173]:
```suggestion
安装完成后，你可以在运行 Docker 的主机上执行 `curl` 命令访问 Admin API，根据返回数据判断 APISIX 是否成功启动。
```

File: docs/zh/latest/getting-started.md [hunk 4]
```diff
@@ -181,25 +242,9 @@ curl "http://127.0.0.1:9080/apisix/admin/upstreams/1" -H "X-API-KEY: edd1c9f0343
 }'
 ```
 
-我们使用 `roundrobin` 作为负载均衡机制，并将 `httpbin.org:80` 设置为我们的上游服务，其 ID 为 `1`。更多字段信息，请参阅 [Admin API](./admin-api.md)。
+该上游配置与上一节配置在路由中的上游相同。同样使用了 `roundrobin` 作为负载均衡机制以及设置了 `httpbin.org:80` 为上游服务。为了将该上游绑定到路由我们把 `upstream_id` 设置为 `1`。更多字段信息，请参考 [Admin API](./admin-api.md)。
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:245]:
```suggestion
该上游配置与上一节配置在路由中的上游相同。同样使用了 `roundrobin` 作为负载均衡机制，并设置了 `httpbin.org:80` 为上游服务。为了将该上游绑定到路由，这里需要把 `upstream_id` 设置为 `1`。更多字段信息，请参考 [Admin API](./admin-api.md)。
```

File: docs/zh/latest/getting-started.md [hunk 5]
```diff
@@ -210,114 +255,34 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
...
-    },
-    "key-auth": {}
-  },
-  "upstream_id": "1"
-}'
-```
+该请求将被 APISIX 转发到 `http://httpbin.org:80/anything/foo?arg=10`。
 
-现在您可以使用以下命令来调用路由：
+## APISIX Dashboard
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:266]:
```suggestion
## 使用 APISIX Dashboard
```

File: docs/zh/latest/getting-started.md [hunk 6]
```diff
@@ -210,114 +255,34 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
...
-- 确保所需的所有端口（**默认的 9080/9443/2379**）未被其他系统/进程使用。
+![Creating a Route with APISIX Dashboard](../../assets/images/list-of-routes.png)
 
-    下面是终止正在侦听特定端口（基于 unix 的系统）的进程的命令。
+想要了解更多关于 APISIX Dashboard 的信息，请参考 [APISIX Dashboard 文档](/docs/dashboard/USER_GUIDE)。
 
-    ```bash
-    sudo fuser -k 9443/tcp
-    ```
+## 后续步骤
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:284]:
```suggestion
## 总结
```

File: docs/zh/latest/getting-started.md [hunk 7]
```diff
@@ -210,114 +255,34 @@ curl "http://127.0.0.1:9080/apisix/admin/routes/1" -H "X-API-KEY: edd1c9f034335f
...
 
-- 如果 Docker 容器持续不断地重启或失败，请登录容器并观察日志以诊断问题。
+完成上述步骤后，APISIX 就可以正常运行了。现在你可以为 APISIX 添加一些插件，来实现身份验证、安全性、限流限速和可观测性等功能。详细信息请参考[插件市场](/plugins)。
```
Comment (SylviaBABY) [inline docs/zh/latest/getting-started.md:286]:
```suggestion
完成上述步骤后，APISIX 就可以正常运行了。如果想利用 APISIX 实现身份验证、安全性、限流限速和可观测性等功能，可通过添加插件实现。各类插件的详细信息请参考[插件市场](/plugins)。
```

**Comment by SylviaBABY** [comment]
the table preview is broken, need check again.
image

**Comment by SylviaBABY** [comment]
I think we could add the step to mark "those are all necessary&indispensable steps", not after installing APISIX it's over(just suggestion)
image

File: docs/zh/latest/getting-started.md [hunk 8]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+- 如何使用 Docker 安装并运行 APISIX。
+- 如何使用 Admin API 创建第一个路由并配置上游。
+- 如何使用 APISIX Dashboard。
+- 如何寻求帮助。
 
-我们将以下面的 `echo` 端点为例，它将返回我们传递的参数。
+## Apache APISIX 是什么？
 
-**请求内容**
+Apache APISIX 是一个开源的云原生 API 网关，作为 API 网关，它兼具动态、实时、高性能等特点，提供了负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。你可以使用 Apache APISIX 来处理传统的南北向流量，以及服务间的东西向流量，也可以当做 K8s Ingress controller 来使用。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:45]:
```suggestion
Apache APISIX 是 Apache 软件基金会下的云原生 API 网关，它兼具动态、实时、高性能等特点，提供了负载均衡、动态上游、灰度发布（金丝雀发布）、服务熔断、身份认证、可观测性等丰富的流量管理功能。我们可以使用 Apache APISIX 来处理传统的南北向流量，也可以处理服务间的东西向流量。同时，它也支持作为 [K8s Ingress Controller](https://github.com/apache/apisix-ingress-controller) 来使用。
```

File: docs/zh/latest/getting-started.md [hunk 9]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+
+### APISIX 的功能
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:47]: We don't need to mention `APISIX` always here :) 

```suggestion
### 主要特性
```

File: docs/zh/latest/getting-started.md [hunk 10]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+
+- 多平台支持：APISIX 提供了多平台解决方案，它不但支持裸机运行，也支持在 Kubernetes 中使用，还支持与 AWS Lambda、Azure Function、Lua 函数和 Apache OpenWhisk 等云服务集成。
+- 全动态能力：APISIX 支持热加载，这意味着你不需要重启服务就可以更新 APISIX 的配置。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:50]:
```suggestion
- 全动态能力：APISIX 支持热加载，这意味着你不需要重启服务就可以更新 APISIX 的配置。请访问[为什么 Apache APISIX 选择 Nginx + Lua 这个技术栈？](https://apisix.apache.org/zh/blog/2021/08/25/why-apache-apisix-chose-nginx-and-lua/)以了解实现原理。
```

File: docs/zh/latest/getting-started.md [hunk 11]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+- 精细化路由：APISIX 支持使用 NGINX 所有的内置变量做为路由的条件，你可以自定义匹配函数来过滤路由。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:51]:
```suggestion
- 精细化路由：APISIX 支持使用 [NGINX 内置变量](http://nginx.org/en/docs/varindex.html)做为路由的匹配条件，你可以自定义匹配函数来过滤路由。
```

Comment (juzhiyuan) [reply docs/zh/latest/getting-started.md:51]: `过滤路由` or `过滤请求`?

File: docs/zh/latest/getting-started.md [hunk 12]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+- 运维友好：APISIX 因其对 DevOps 团队的操作友好而闻名。它可以与以下工具和平台集成：[HashiCorp Vault](./plugins/jwt-auth.md#usage-with-hashicorp-vault)、[Zipkin](./plugins/zipkin.md)、[Apache SkyWalking](./plugins/skywalking.md)、[Consul](./discovery/consul_kv.md)、[Nacos](./discovery/nacos.md)、[Eureka](./discovery.md)。通过 [APISIX Dashboard](/docs/dashboard/USER_GUIDE)，运维人员可以通过友好且直观的 UI 配置 APISIX。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:52]: `APISIX 因其对 DevOps 团队的操作友好而闻名` 🤣🤣🤣

File: docs/zh/latest/getting-started.md [hunk 13]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+- 多语言插件支持：APISIX 支持多种开发语言进行插件开发，开发人员可以选择擅长语言的 SDK 开发自定义插件。
+
+## 主要概念
+
+下图为 APISIX 的架构：
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:57]:
```suggestion
下图为 Apache APISIX 的架构：
```

File: docs/zh/latest/getting-started.md [hunk 14]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+
+![flow-software-architecture](https://raw.githubusercontent.com/apache/apisix/master/docs/assets/images/flow-software-architecture.png)
+
+下表是本文涉及到的 APISIX 的主要概念和组件：
+
+| 概念/组件    | 描述                                                                                             |
+|-------------|--------------------------------------------------------------------------------------------------|
+| Route       | 通过路由定义规则来匹配客户端请求，根据匹配结果加载并执行相应的插件，最后把请求转发给到指定的上游应用。  |
+| Upstream    | 上游的作用是按照配置规则对服务节点进行负载均衡，它的地址信息可以直接配置到路由或服务上。               |
+| Admin API   | 用户通过该组件可以控制 APISIX 实例的 API。                                                         |
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:67]:
```suggestion
| Admin API   | 用户通过 AdminAPI 可以控制 APISIX 实例。                                                         |
```

File: docs/zh/latest/getting-started.md (hunk 1 above)
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:75]: How about converting this NOTE to the `INFO` mark? (Not in this PR) AFAIK, some users don't know what's `httpbin` is and they will ignore this part usually 🤣

cc @hf400159 @navendu-pottekkat

File: docs/zh/latest/getting-started.md [hunk 15]
```diff
@@ -21,25 +26,61 @@ title: 快速入门指南
...
+
+在开始使用 APISIX 之前，请确保你已经安装以下应用：
+
+- [Docker](https://www.docker.com/) 和 [Docker Compose](https://docs.docker.com/compose/)。
+- [curl](https://curl.se/docs/manpage.html) 用于测试 API。你也可以使用 [Hoppscotch](https://hoppscotch.io/) 之类的工具。
+- 本文使用的上游服务是 [httpbin.org](https://httpbin.org)，你可以使用它进行测试。它是一个返回服务，这意味着它将返回我们在请求中传递的参数。
+
+**请求内容：**
 
 请求 URL 由以下这些参数构成：
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:79]:
```suggestion
请求 URL 由以下参数构成：
```

File: docs/zh/latest/getting-started.md [hunk 16]
```diff
@@ -68,51 +109,75 @@ curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
 }
 ```
 
-## 前提条件
+## 安装 APISIX
 
-- 已安装 [Docker Compose 组件](https://docs.docker.com/compose/)。
+在本文中我们将使用 Docker 安装 APISIX 并启用 [Admin API](./admin-api.md)。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:114]:
```suggestion
我们将使用 Docker 安装 Apache APISIX 并启用 [Admin API](./admin-api.md)。
```

File: docs/zh/latest/getting-started.md [hunk 17]
```diff
@@ -68,51 +109,75 @@ curl --location --request GET "http://httpbin.org/get?foo1=bar1&foo2=bar2"
...
-#    Please refer to https://docusaurus.io/docs/next/markdown-features/admonitions
-#    for more detail.
-#
--->
+```bash
+git clone https://github.com/apache/apisix-docker.git
+cd apisix-docker/example
+```
+
+现在你可以 `docker-compose` 启动 APISIX。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:123]:
```suggestion
现在你可以通过 `docker-compose` 启动 APISIX 实例。
```

File: docs/zh/latest/getting-started.md [hunk 18]
```diff
@@ -126,17 +191,13 @@ curl "http://127.0.0.1:9080/apisix/admin/services/" -H 'X-API-KEY: edd1c9f034335
...
-您现在已经拥有一个运行中的 Apache APISIX 实例了！接下来，让我们来创建一个路由（Route）。
-
-### 工作原理
+## 创建路由
 
-Apache APISIX 提供了强大的 [Admin API](./admin-api.md) 和 [Dashboard](https://github.com/apache/apisix-dashboard) 可供我们使用。在本文中，我们使用 Admin API 来做演示。
+现在你已经拥有一个正在运行的 APISIX 实例了！接下来，让我们来创建一个路由。
 
-我们可以创建一个 [Route](./terminology/route.md) 并与上游服务（通常也被称为 [Upstream](./terminology/upstream.md) 或后端服务）绑定，当一个 `请求（Request）` 到达 Apache APISIX 时，Apache APISIX 就会明白这个请求应该转发到哪个上游服务中。
+APISIX 提供了强大的 [Admin API](./admin-api.md) 和 [Dashboard](https://github.com/apache/apisix-dashboard) 供用户使用。在下述示例中，我们将使用 Admin API 创建一个 [Route](./terminology/route.md) 并与 [Upstream](./terminology/upstream.md) 绑定，当一个请求到达 APISIX 时，APISIX 会将请求转发到指定的上游服务中。
```
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:198]: One suggestion: we need to unify when to use `Route`, and when to use `路由`。

Comment (juzhiyuan) [reply docs/zh/latest/getting-started.md:198]: I would prefer to always use `Route, Upstream, Service`, even in Chinese, as they're more straightforward.

Comment (juzhiyuan) [reply docs/zh/latest/getting-started.md:198]: cc @navendu-pottekkat

File: docs/zh/latest/getting-started.md (hunk 4 above)
Comment (juzhiyuan) [inline docs/zh/latest/getting-started.md:245]: People will get confused it's string or number 1 :)

**Review by juzhiyuan — COMMENTED** [review]
Most LGTM

File: docs/zh/latest/getting-started.md (hunk 16 above)
Comment (hf400159) [reply docs/zh/latest/getting-started.md:114]: Here we don't need to use Apache APISIX. Please refer to [语言](https://apisix.apache.org/zh/docs/general/documentation-style-guide#%E8%AF%AD%E8%A8%80)

File: docs/zh/latest/getting-started.md (hunk 18 above)
Comment (hf400159) [reply docs/zh/latest/getting-started.md:198]: > I would prefer to always use `Route, Upstream, Service`, even in Chinese, as they're more straightforward.

`Route` is used when Route has a representation concept, if it is a representation entity, such as creating a Route. will use `路由`. I will update the guide.

Comment (navendu-pottekkat) [reply docs/zh/latest/getting-started.md:198]: I'm not sure what Chinese readers prefer but whichever one we choose, it should be consistent. It should be also added to the style guide for reference in the future.

**Review by SylviaBABY — APPROVED** [review]

**Comment by SylviaBABY** [comment]
LGTM, but the preview link not the latest content

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**